### PR TITLE
Changed the pom for activecollab,amazons3,amazonsdb,amazonses and callrail

### PR DESCRIPTION
**Pull request by katheesR**

### CODE REVIEW

File: activecollab/activecollab-connector/activecollab-connector-1.0.0/org.wso2.carbon.connector/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
         <groupId>org.wso2.carbon</groupId>
```
Comment (prabathabey) [inline activecollab/activecollab-connector/activecollab-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: License header is missing.

File: callrail/callrail-connector/callrail-connector-1.0.0/org.wso2.carbon.connector/pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 		<groupId>org.wso2.carbon</groupId>
```
Comment (prabathabey) [inline callrail/callrail-connector/callrail-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: License header is missing

File: insightly/insightly-connector/insightly-connector-1.0.0/org.wso2.carbon.connector/pom.xml [hunk 3]
```diff
@@ -6,7 +6,7 @@
         <groupId>org.wso2.carbon</groupId>
```
Comment (prabathabey) [inline insightly/insightly-connector/insightly-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: License header is missing.

File: pom.xml [hunk 4]
```diff
@@ -25,28 +25,90 @@
...
-        <!--<module>evernote/evernote-connector/evernote-connector-1.0.0</module>-->
-        <!--<module>campaignmonitor/cm-connector/cm-connector-1.0.0/org.wso2.carbon.connector</module>-->
-        <!--<module>concur/concur-connector/concur-connector-1.0.0/org.wso2.carbon.connector</module>-->
-        <!--<module>quickbooks/quickbooks-connector/quickbooks-connector-1.0.0/org.wso2.carbon.connector.quickbooks</module>-->
-        <!--<module>stripe</module>-->
-        <!--<module>xero/xero-connector/xero-connector-1.0.0/org.wso2.carbon.connector</module>-->
-        <!--<module>yelp/yelp-connector/yelp-connector-1.0.0</module>-->
-        <!--<module>zohocrm/zohocrm-connector/zohocrm-connector-1.0.0/org.wso2.carbon.connector</module>-->
-        <!--<module>mailchimp/mailchimp-connector/mailchimp-connector-1.0.0/org.wso2.carbon.connector</module>-->
+        <!--
```
Comment (prabathabey) [inline pom.xml]: Wrong license header is used.

File: activecollab/activecollab-connector/activecollab-connector-1.0.0/org.wso2.carbon.connector/pom.xml (hunk 1 above)
Comment (katheesR) [inline activecollab/activecollab-connector/activecollab-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: Fixed in the PR - https://github.com/wso2/esb-connectors/pull/340

File: callrail/callrail-connector/callrail-connector-1.0.0/org.wso2.carbon.connector/pom.xml (hunk 2 above)
Comment (katheesR) [inline callrail/callrail-connector/callrail-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: Fixed in the PR - https://github.com/wso2/esb-connectors/pull/340

File: insightly/insightly-connector/insightly-connector-1.0.0/org.wso2.carbon.connector/pom.xml (hunk 3 above)
Comment (katheesR) [inline insightly/insightly-connector/insightly-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: Fixed in the PR - https://github.com/wso2/esb-connectors/pull/340

File: pom.xml (hunk 4 above)
Comment (katheesR) [inline pom.xml]: Fixed in the PR - https://github.com/wso2/esb-connectors/pull/340

**Pull request merged**